### DISCUSSION TITLE
not all raw files contribute to library?

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#421] [Q&A]
I ran a search with DIANN1.8.1 on 96 raw files and a library in smart profiling mode. 

All 96 raw files were analysed in FirstPass and SecondPass search. 
However, only 84 raw files contributed to enhancing the library in the first cross-run analysis between FirstPass and SecondPass. 

For instance raw file with POS-A02 is missing:
![image](https://user-images.githubusercontent.com/98888664/176014319-093b502c-434f-455a-aff2-b06dedb52f3e.png)

control, A02 is loaded raw file:
![image](https://user-images.githubusercontent.com/98888664/176014509-01c5167f-77c2-4400-903e-bcc40889f599.png)

Why is that?
I loaded the raw files from a local drive, so there certainly was no skipping due to a network issue. 
[DA-O1.1_PL01_DA-SER1.15_dia raw_on diskD_.log.txt](https://github.com/vdemichev/DiaNN/files/8994801/DA-O1.1_PL01_DA-SER1.15_dia.raw_on.diskD_.log.txt)

Is there a filter leading to that behaviour ? (e.g. only letting runs with high-scoring PSMs etc. improve the library?)

**Comment by vdemichev** [comment]
That's OK and is by design. For example, if some samples are blanks, or just don't contain good speactra for the peptides of interest, no need to load them.
